troubleshooting/ETW/Readme.md

```diff
@@ -122,7 +122,8 @@ As of version 2.18.0, this SDK ships a "self-diagnostics feature" which captures
 
 The self-diagnostics feature can be enabled/changed/disabled while the process is running.
 The SDK will attempt to read the configuration file every 10 seconds, using a non-exclusive read-only mode.
-The SDK will create or overwrite a file with new logs according to the configuration. 
+Each time this configuration file changes, a new log file will be created according to the configuration. 
```

Comment:
A new log file will not be created if only `LogLevel` is changed. As the log file related properties like log location and file size did not change.

Reply:
And it won't create log file and actually close the file if the config file becomes an invalid file (typo, missing fields, etc.).
Can you fix this before merging?